Essentials/Commands/AdminModule.cs

```diff
@@ -49,6 +49,24 @@ public void PlayerCount(int count = -1)
             Context.Respond($"Max player count: {MyMultiplayer.Static.MemberLimit}. Current online players: {MyMultiplayer.Static.MemberCount - 1}");
         }
 
+        [Command("playerlist", "Lists current players on the server")]
+        [Permission(MyPromoteLevel.Admin)]
+        public void ListPlayers()
+        {
+            if(MySession.Static.Players.GetOnlinePlayerCount() == 0)
+            {
+                Context.Respond("No players online");
+                return;
+            }
+            StringBuilder sb = new StringBuilder();
+            foreach(var player in MySession.Static.Players.GetOnlinePlayers())
+            {
+                sb.Append($"{player.DisplayName}, ");
+            }
+            Context.Respond(sb.ToString());
```

Comment:
I'd recommend using a dialog here. You can send a DialogMessage through ModCommunication.